bpf/node_config.h

```diff
@@ -35,6 +35,10 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
  #define SECCTX_FROM_IPCACHE fetch_u32(SECCTX_FROM_IPCACHE)
 #endif
 
+#ifndef IS_L3_DEV
+#define IS_L3_DEV(ifindex) false
+#endif
```

Comment:
This is already defined on line 213 of this same file.

Reply:
In the first place there wasn't such definition until I rebased the upstream. Let me delete the duplicate macro.

Reply:
The duplicated macro has been removed and I'll resolve this.